external/esp_idf_port/esp32/spi_ram/spiram_psram.c

```diff
@@ -33,16 +33,12 @@
  Driver bits for PSRAM chips (at the moment only the ESP-PSRAM32 chip).
 */
 
-//#include "sdkconfig.h"
 #include "string.h"
 #include "esp_attr.h"
-#include "esp_err.h"
-//#include "esp_types.h"
-//#include "esp_log.h"
+#include <arch/chip/esp_err.h>
```

Comment:
unnecessary here when it's already include in the h file.

Reply:
I will delete this line, thanks.